spec/features/shelter_reviews/new_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "new shelter review page" do
+  it "can create a new review using a form" do
+    shelter_1 = Shelter.create(name: "Happy Puppies", address: "55 Street St", city: "Danger Mountain", state: "UT", zip: "80304")
+
+    visit "/shelters/#{shelter_1.id}/reviews/new"
+
+    fill_in :title, with: "What a great place!"
+    select "5", from: :rating
+    fill_in :content, with: "This is a Review"
+    fill_in :image, with: "image.jpeg"
+
+    click_button "Submit Review"
+
+    expect(current_path).to eq("/shelters/#{shelter_1.id}")
+
+    expect(page).to have_content("What a great place!")
+    expect(page).to have_content("5")
+    expect(page).to have_content("This is a Review")
+    # expect(page).to have_css("img[src*='image.jpeg']")
```

Comment:
This test worked for the old html but I can't figure out how to test the new image_tag

Reply:
I verified the functionality on the local host

Reply:
I did some looking around, and I cannot find an answer for this. Perhaps we can ask this question in our Slack channel.

Reply:
After our pairing earlier, I'm convinced that this should work the same way that all the other tests for presence of image work! I'm gonna plug away at this a little longer before merging.

Reply:
Update: I figured out why it doesn't work. So that's a good start. Leaving it as is, but added it as an issue to our project board.